### DISCUSSION TITLE
Block a computer

### DISCUSSION THREAD

**Discussion by mrkacg1** [Ylianst/MeshCentral#5716] [Q&A]
Hello. I have a situation:

I installed meshcentral on a Mac and can no longer monitor it. I have tried uninstalling it but this does not work. I no longer have access to said device and I want to know if there is a way to block it from continuing to register on the meshcentral device list.

**Comment by dinger1986** [comment]
Can you get into the terminal and execute an uninstall via the command line?

**Comment by si458** [comment]
If the device appears in ur Web UI, you could try going into it, then console tab and type `uninstallagent`
Or
Into the device, general tab, actions, then `uninstall agent`

**Comment by mrkacg1** [comment]
hello. It's not possible, it's the first thing you try:



I get this error in console:



uncaughtException1: Error: => EventEmitter.emit(): Event dispatch for 'Command' on 'MeshAgent' threw an exception: LaunchDaemon (undefined) NOT FOUND in method 'handleServerCommand()'

















--

Gracias. 

Quedo atento a sus comentarios, ¡Saludos!.










---- El Sat, 20 Jan 2024 08:48:17 -0500, Simon Smith ***@***.***> escribió ----





If the device appears in ur Web UI, you could try going into it, then console tab and type uninstallagent
 Or
 Into the device, general tab, actions, then uninstall agent
